### PR TITLE
Fix artifact references

### PR DESCRIPTION
**Pull request by bkaney**
* Fix artifact reference
* Also added some future ballot work in non-building directory

### CODE REVIEW

File: input/pages/operations.md [hunk 1]
```diff
@@ -41,15 +41,14 @@ These operations are defined to support artifact authoring capabilities expected
...
-| [Manifestable][4]                   | Defines parameters used to provide a version manifest                          |
-| [Pageable][5]                       | Defines parameters for an operation with pageable options                      |
-| [Artifact Endpoint Configurable][6] | Defines parameters to provide endpoints to resolve canonical artifacts         |
-| Data Configurable (Roadmap)         | Defines parameters to specify how to resolve data requirements                 |
+| [Artifact Version Bindable][3]      | Defines parameters used to specify missing versions in all artifact references                                         |
+| [Manifestable][4]                   | Defines parameters used to provide a version manifest                                                                  |
+| [Pageable][5]                       | Defines parameters for an operation with pageable options                                                              |
+| [Artifact Endpoint Configurable][6] | Defines parameters to provide endpoints to resolve canonical artifacts                                                 |
```
Comment (bkaney) [inline input/pages/operations.md:51]:
```suggestion
| [Artifact Endpoint Configurable][6] | Defines parameters to provide endpoints to resolve canonical artifacts                                                 |
| Data Configurable (Roadmap)         | Defines parameters to specify how to resolve data requirements                 |
| Parameterizable (Roadmap)            | Defines parameters passed to logic evaluation (e.g. CQL parameters)         |
```

Comment (bkaney) [inline input/pages/operations.md:51]:
```suggestion
| [Artifact Endpoint Configurable][6] | Defines parameters to provide endpoints to resolve canonical artifacts                                                 |
| Data Configurable (Roadmap)         | Defines parameters to specify how to resolve data requirements                 |
| Parameterizable (Roadmap)            | Defines parameters passed to logic evaluation (e.g. CQL parameters)         |
```

**Comment by bkaney** [comment]
@brynrhodes  - can you take a look at this?

**Comment by bkaney** [comment]
@brynrhodes  - can you take a look at this?

**Review by brynrhodes — APPROVED** [review]
QA: fixing implementation of artifact reference pattern

**Pull request merged**